java/ingest/ingest-runner/src/test/java/sleeper/ingest/impl/IngestCoordinatorCommonIT.java

```diff
@@ -132,6 +136,7 @@ private StateStore createStateStore(Schema schema) {
     public void shouldWriteRecordsCorrectly(TestIngestType ingestType)
             throws StateStoreException, IOException, IteratorException {
         // Given
+        tableProperties.setEnum(INGEST_FILE_WRITING_STRATEGY, ONE_FILE_PER_LEAF);
```

Comment:
It would be nice if we could do this setup once rather than in every test. We could set that up with nested test classes for each strategy?

Reply:
Thanks, I've moved the tests out into a new class `IngestCoordinatorFileWritingStrategyIT`